components/integrations_marketplace/index.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+
+import {installPluginFromUrl} from 'mattermost-redux/actions/admin';
+import {getMarketplacePlugins} from 'mattermost-redux/actions/plugins';
+import {getMarketplaceInstalledPlugins} from 'mattermost-redux/selectors/entities/plugins';
+
+import {isModalOpen} from 'selectors/views/modals';
+import {ModalIdentifiers} from 'utils/constants';
+
+import {closeModal} from 'actions/views/modals';
+
+import MarketplaceModal from './marketplace_modal.jsx';
+
+function mapStateToProps(state) {
+    return {
+        show: isModalOpen(state, ModalIdentifiers.INTEGRATIONS_MARKETPLACE),
+        installedPlugins: getMarketplaceInstalledPlugins(state),
+        marketplacePlugins: state.entities.plugins.marketplacePlugins,
+    };
+}
+
+function mapDispatchToProps(dispatch) {
+    return {
+        actions: bindActionCreators({
+            closeModal: () => closeModal(ModalIdentifiers.INTEGRATIONS_MARKETPLACE),
+            installPluginFromUrl,
```

Comment:
should this be removed? doesn't look like it's being used.

Reply:
Good catch 👍Leftover from refactoring